proof/refine/ARM/Untyped_R.thy

```diff
@@ -4331,7 +4331,7 @@ lemma resetUntypedCap_corres:
   apply (frule if_unsafe_then_capD'[OF ctes_of_cte_wpD], clarsimp+)
   apply (frule(1) descendants_range_ex_cte'[OF empty_descendants_range_in' _ order_refl],
     (simp add: isCap_simps)+)
-  apply (intro conjI impI; clarsimp)
+  apply (auto simp: descendants_range_in'_def valid_untyped'_def)
```

Comment:
does this imply the previous frule step did not succeed in discharging the side-conditions? feels strange this is so different now

Reply:
I unfortunately know very little about these properties, so I'm not sure exactly what's going on here: I again simply copied what was done on the `rt` branch. None of those `frule`s can be removed, so I imagine they're doing something. I can look into this some more if it seems like there might be an issue

Reply:
I didn't mean removing the frules, I was curious why we needed the switch to auto with args with what seems like an unrelated change. It doesn't make a big difference, and if `rt` has it anyway, shrugs.